framework/src/org/checkerframework/common/util/count/AnnotationsCounter.java

```diff
@@ -26,34 +22,31 @@
  */
 @SupportedAnnotationTypes("*")
 @SupportedSourceVersion(SourceVersion.RELEASE_8)
-public class AnnotationsCounter extends AbstractProcessor {
+public class AnnotationsCounter extends AbstractTypeProcessor {
     final Map<Name, Integer> annotationCount = new HashMap<Name, Integer>();
 
     protected void incrementCount(Name annoName) {
         if (!annotationCount.containsKey(annoName)) {
-            annotationCount.put(annoName, 2);
+            annotationCount.put(annoName, 1);
```

Comment:
I didn't see a reason why counting should start at 2. With the small examples I tried 1 made more sense.